src/npt/arch/aarch64.rs

```diff
@@ -106,7 +119,11 @@ impl From<DescriptorAttr> for MappingFlags {
 impl From<MappingFlags> for DescriptorAttr {
     fn from(flags: MappingFlags) -> Self {
         let mut attr = if flags.contains(MappingFlags::DEVICE) {
-            Self::from_mem_type(MemType::Device)
+            if flags.contains(MappingFlags::UNCACHED) {
```

Comment:
Can we just convert `MappingFlags` to `DescriptorAttr` directly?

Why using `from_mem_type` here, it seems redundant

Reply:
I also think that it can be directly converted here. 
The `from_mem_type` writen by @erquation314, maybe it would be more concise?

Reply:
Never mind, just follow the old code style.

@aarkegz  when do we start to work on axaddrspace?